rllib/examples/multi_agent/restore_1_of_n_agents_from_checkpoint.py

```diff
@@ -0,0 +1,151 @@
+"""Simple example of loading module weights for 1 of n agents from checkpoint.
```

Comment:
nit: Let's not use "simple".

"An example script showing how to load RLModule weights for 1 out of n agents from a checkpoint" ?

Reply:
Yup, not simple for everyone lol. I know what you mean, its actually quite some complexity to make this possible in MA scenarios - and so powerful.